Makefile

```diff
@@ -502,7 +502,7 @@ clean:
 
 pkg/apis/workflow/v1alpha1/openapi_generated.go: $(GOPATH)/bin/openapi-gen $(TYPES)
 	[ -e v3 ] || ln -s . v3
-	openapi-gen \
+	${GOPATH}/bin/openapi-gen \
```

Comment:
$(GOPATH)?

Reply:
I used curly brances to be consistent with other similar places in Makefile, e.g. `${GOPATH}/bin/go-to-protobuf`. Would you like me to change all of them to `()`?

Reply:
Just replaced all the `{}` by `()`.